tests/test_allowed_recipients_builder.py

```diff
@@ -182,62 +181,76 @@ def test_deploy_allowed_recipients_registry(
     accounts,
     stranger,
     agent,
+    ldo,
+    usdc,
     evm_script_executor,
     AllowedRecipientsRegistry,
 ):
     limit = 1e18
     period = 1
     recipients = [accounts[3], accounts[4]]
+    tokens = [ldo, usdc]
     titles = ["account 3", "account 4"]
     spentAmount = 1e10
 
-    tx = allowed_recipients_builder.deployAllowedRecipientsRegistry(
+    tx_recipients = allowed_recipients_builder.deployAllowedRecipientsRegistry(
         limit, period, recipients, titles, spentAmount, True, {"from": stranger}
     )
 
-    registry_address = tx.events["AllowedRecipientsRegistryDeployed"][
+    tx_tokens = allowed_recipients_builder.deployAllowedTokensRegistry(tokens, {"from": stranger})
```

Comment:
Move checks for allowed tokens registry in a standalone test. Similar to "test_deploy_allowed_recipients_registry", check that:
- roles were granted correctly
- builder doesn't have any roles granted
- all tokens were added